setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.9',
+        'django>=1.4,<1.10',
```

Comment:
This restriction should be removed entirely. Since we always run test on django master, we should leave this unbounded so we can always be apprised of failures (the master branch of Django always bumps the version up to the next-to-be released version; see https://github.com/django/django/blob/master/django/__init__.py#L5 which currently is already listing the version as 1.10.alpha; this unnecessary restriction means that "django master" builds are actually running on Django 1.9)